backend/cms/models/page.py

```diff
@@ -67,14 +67,14 @@ def languages(self):
     def get_translation(self, language_code):
         return self.page_translations.filter(
             language__code=language_code
-        ).order_by('-version').first()
```

Comment:
We don't need this ordering anymore, because I addded the default order to the meta class

Reply:
Good to know.